tensorflow/lite/delegates/webnn/add_test.cc

```diff
@@ -0,0 +1,935 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Let's use Copyright 2022 for all new files.
```suggestion
/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
```